### PR TITLE
Add did:key with 2020 test suite from Digital Bazaar.

### PR DESCRIPTION
**Pull request by msporny**

### CODE REVIEW

File: packages/did-core-test-server/suites/implementations/did-key-2020-db.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    "application/did+ld+json"
+  ],
+  "dids": ["did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH"],
+  "didParameters": {},
+  "did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH": {
+    "application/did+ld+json": {
+      "didDocument": {
+        "@context": [
+          "https://www.w3.org/ns/did/v1",
+          "https://w3id.org/security/suites/ed25519-2020/v1",
```
Comment (OR13) [inline packages/did-core-test-server/suites/implementations/did-key-2020-db.json:15]: nice!

Comment (peacekeeper) [reply packages/did-core-test-server/suites/implementations/did-key-2020-db.json:15]: +1, agree with this approach

**Comment by msporny** [comment]
@peacekeeper @OR13 -- this is ready for final review. It uses the new format that separates the data model properties from the representation specific entries. 

I'm fairly certain we're ready for other DID Methods to do their implementations now.

**Review by OR13 — APPROVED** [review]

**Review by peacekeeper — APPROVED** [review]
Looks good.. Details may have to be updated later if the input data structure for tests changes.

**Pull request merged**